tests/suites/test_suite_psa_crypto_pake.function

```diff
@@ -39,6 +40,10 @@ typedef enum {
     ERR_INJECT_ROUND2_SERVER_KEY_SHARE,
     ERR_INJECT_ROUND2_SERVER_ZK_PUBLIC,
     ERR_INJECT_ROUND2_SERVER_ZK_PROOF,
+    ERR_INJECT_EXTRA_OUTPUT,
+    ERR_INJECT_EXTRA_INPUT,
+    ERR_INJECT_EXTRA_OUTPUT_AT_END,
+    ERR_INJECT_EXTRA_INPUT_AT_END,
```

Comment:
With the new test function argument we do not those two I'd say.
```suggestion
```

Reply:
Removing these tests prevents full coverage of `psa_jpake_epilogue()` (result below). I think I could remove just one of them if that is beneficial?

![image](https://github.com/Mbed-TLS/mbedtls/assets/70948878/a4f321d4-29d1-4eee-8002-031941d7e6d8)